subprojects/gradle/runner/shared/src/main/kotlin/com/avito/runner/CommandLineExecutor.kt

```diff
@@ -28,13 +28,14 @@ interface CommandLineExecutor {
             output: File?
         ): Observable<ProcessNotification> = Observable.create(
             { emitter ->
-                val outputFile: File = when {
-                    output == null || output.isDirectory -> {
-                        prepareOutputFile(output, true)
+                val outputFile: File? = when {
+                    output == null -> null
```

Comment:
I supposed if the output file is null that we don't need to create ours because it won't be used